go/script/asdf_bootstrap.sh

```diff
@@ -0,0 +1,94 @@
+#/bin/zsh
+
+# ==================================================================================
+# This script will attempt to setup asdf / direnv on a given system.
+# I have tested Linux but not MacOS. 
+#
+# It should install standard asdf plugins based on what you list in .tool-versions
+# ==================================================================================
+
+init_os_specific_vars() {
+	OS=`uname -s`
+	case "$OS" in
+		"Darwin") 	INSTALL_CMD="brew install"
+				ASDF_INSTALL_PKG="asdf"
+				;;
+		"Linux")  	ASDF_INSTALL_PKG="asdf-vm"
+				. "/etc/lsb-release"
+				echo "Distro is ${DISTRIB_ID} ($DISTRIB_RELEASE)"
+				case "$DISTRIB_ID" in
```

Comment:
This might be easier if Linux users just installed brew, but I looked at the janky brew setup on Linux and it made me :sob: